First Commit for Contact

jeffreyrivera0318 opened the pull request:
1. Created and edited inquiries table
2. Created and edited inquiry model
3. Created and edited inquiry request
4. Created and edited inquiry controller
5. Created routes for inquiry
6. Edited the following blade file: contact, confirm

jeffreyrivera0318 committed:
First Commit for Contact

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,71 @@
+{
+    /**
+     * Show the form
+     * @return Response
```
change

```
* Show contact form
* @return view
```

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,71 @@
+     */
+    public function index()
+    {
+        return view('adsist.contact.contact');
```
delete parent adsist folder
return view('contact.contact_form');

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,71 @@
+    {
+        $data = $request->session()->get('inquiry');
+
+        return view('adsist.contact.contact_confirm.confirm', compact('data'));
```
change
return view('adsist.contact.contact_confirm', compact('data'));

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,71 @@
+    // Clear the session data
+    $request->session()->forget('inquiry');
+
+    return view('adsist.contact.contact_complete.complete');
```
change
return view('adsist.contact.contact_complete');

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,71 @@
+     * @param  Request $request
+     * @return Response
+     */
+    public function save(Request $request)
```
remove $request params is unused

ChronoDevs commented on app/Models/inquiry.php:
```diff
@@ -0,0 +1,19 @@
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class inquiry extends Model
```
Model should be PascalCase

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,71 @@
+use App\Models\inquiry;
+use Illuminate\Http\Request;
+
+class inquiryController extends Controller
```
Controller should be PascalCase

jeffreyrivera0318 committed:
Second Commit for Contact

jeffreyrivera0318 committed:
transfer folder

jeffreyrivera0318 committed:
third commit

jeffreyrivera0318 committed:
transferred the blade file

ChronoDevs commented on resources/views/mail/layout.blade.php:
```diff
@@ -0,0 +1,27 @@
+        © {{config('app.name')}}. All rights reserved.
+    @endcomponent
+@endslot
+@endcomponent
```
add last break line

ChronoDevs commented on resources/views/contact/contact_confirm/confirm.blade.php:
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="ja">
```
Please extends app.blade

ChronoDevs commented on resources/views/contact/contact_complete/complete.blade.php:
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```
Please extends app.blade

ChronoDevs commented on resources/views/contact/contact.blade.php:
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="ja">
```
Please extends app.blade

ChronoDevs commented on resources/views/adsist/contact/contact_confirm/confirm.blade.php:
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="ja">
```
Please extends app.blade

ChronoDevs replied:
and delete adsist/contact/contact_confirm parent folder just views/contact

ChronoDevs commented:
Please fix the conflict. Thank you

jeffreyrivera0318 committed:
Added app.blade, adjust layout.blade

ChronoDevs commented on resources/views/contact/contact.blade.php:
```diff
@@ -0,0 +1,52 @@
+          </form>
+        </div>
+      </div>
+@stop   
```
use @endsection

ChronoDevs commented on resources/views/contact/contact_complete/complete.blade.php:
```diff
@@ -0,0 +1,36 @@
+    <script src="../../js/common.js"></script>
+  </body>
+</html>
+@stop
```
@endsection

ChronoDevs commented on resources/views/contact/contact_confirm/confirm.blade.php:
```diff
@@ -0,0 +1,65 @@
+          </form>
+        </div>
+      </div>
+@stop
```
@endsection

jeffreyrivera0318 committed:
Align with the Global app.blade

ChronoDevs commented on resources/views/contact/contact_complete/complete.blade.php:
```diff
@@ -0,0 +1,60 @@
+@extends('layout.app')
```
Please update the file path views/contact/complete.blade.php

ChronoDevs commented on resources/views/contact/contact_confirm/confirm.blade.php:
```diff
@@ -0,0 +1,93 @@
+@extends('layout.app')
```
Please update the file path views/contact/confirm.blade.php

jeffreyrivera0318 committed:
Updated file path for the blade file for contact

jeffreyrivera0318 committed:
deleted adsist folder

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,73 @@
+<?php
```
use PascalCase for file InquiryController

ChronoDevs commented on app/Models/inquiry.php:
```diff
@@ -0,0 +1,19 @@
+<?php
```
use PascalCase for file Inquiry

ChronoDevs commented on routes/web.php:
```diff
@@ -1,5 +1,6 @@
 <?php
 
+use App\Http\Controllers\inquiryController;
```
use PascalCase for file InquiryController

ChronoDevs commented on database/seeders/InquirySeeder.php:
```diff
@@ -0,0 +1,27 @@
+
+namespace Database\Seeders;
+
+use App\Models\inquiry;
```
please delete if not use
`use App\Models\inquiry;`

ChronoDevs commented on app/Http/Controllers/inquiryController.php:
```diff
@@ -0,0 +1,73 @@
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\InquiryRequest;
```
Please delete if not use
```
use App\Http\Requests\InquiryRequest;
use App\Models\Inquiry;
use App\Models\User;
```

jeffreyrivera0318 committed:
Rename inquiryController.php to InquiryController.php

jeffreyrivera0318 committed:
Rename inquiry.php to Inquiry.php

jeffreyrivera0318 committed:
Update web.php

use PascalCase for InquiryController

jeffreyrivera0318 committed:
Update InquirySeeder.php

Delete App\Models\inquiry

jeffreyrivera0318 committed:
Update InquiryController.php

use App\Http\Requests\LoginRequest;
use App\Models\Inquiry;
use App\Models\User;
